src/main/java/com/minecolonies/coremod/network/messages/SaveScanMessage.java

```diff
@@ -41,7 +49,33 @@ public SaveScanMessage(final NBTTagCompound nbttagcompound, final String storeAt
     @Override
     public void fromBytes(@NotNull final ByteBuf buf)
     {
-        nbttagcompound = ByteBufUtils.readTag(buf);
+        PacketBuffer pb = new PacketBuffer(buf);
+        ByteBufInputStream stream = new ByteBufInputStream(pb);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'stream' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)